### PR TITLE
Use consistent log level for dev service startup messages

### PR DESCRIPTION
**Pull request by michael-simons**
Sorry, I missed this in the original PR, it was a leftover from me debugging things. I would rather have this consistent being `debug`.

### CODE REVIEW

**Comment by gastaldi** [comment]
Shouldn't this be a `warning` instead to let the dev know what happened?

**Comment by gastaldi** [comment]
Or maybe `error/fatal`? Seems too important to be visible only during debug  IMHO

**Comment by michael-simons** [comment]
I took inspiration from the other extensions like mongo and amqp. They look into `debug`. Tbh I’d indeed prefer info.

**Comment by gastaldi** [comment]
Now thinking as a Neo4j user: Considering that this is an important message and the reason why my Neo4J DB is not starting, I'd prefer it to be logged as a warning, no?

**Comment by michael-simons** [comment]
It might be just what I want: I didn’t touch the config and I expect it to just use the running local thing. 

Have to let it sink down until tomorrow. Eventually it’s fine like it is actually.

File: extensions/neo4j/deployment/src/main/java/io/quarkus/neo4j/deployment/Neo4jDevServicesProcessor.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public boolean getAsBoolean() {
             var boldIsReachable = Boolean.getBoolean("quarkus.neo4j.devservices.assumeBoltIsReachable")
                     || boltHandshaker.isBoltPortReachable(Duration.ofSeconds(5));
             if (boldIsReachable && log.isDebugEnabled()) {
```
Comment (gastaldi) [inline extensions/neo4j/deployment/src/main/java/io/quarkus/neo4j/deployment/Neo4jDevServicesProcessor.java:58]: > Have to let it sink down until tomorrow. Eventually it’s fine like it is actually.

Almost :)

```suggestion
            if (boldIsReachable && log.isInfoEnabled()) {
```

Comment (michael-simons) [reply extensions/neo4j/deployment/src/main/java/io/quarkus/neo4j/deployment/Neo4jDevServicesProcessor.java:58]: Lol. Good catch.

**Comment by michael-simons** [comment]
I think you are right. Why? If the configuration has not been changed dev or test mode will overwrite local database, so a warning is appropriate. 

To make this meaningful I however cannot use the boolean supplier and do this explicitly. Thus, the order and level will be
- debug messages if the config has been touched, as it is expected that no dev services should be started
- than, if it would be used but cannot, a warning will be issued
- 
also, i noted that not all the tests did run, thanks to surefire having static issues :(  

If you happy with the changes, I'll squash everything and you do your thing :) Thank you for the value input.

**Review by gastaldi — APPROVED** [review]
LGTM

**Pull request merged**